app/assets/javascripts/listings/templates/listing/_additional_info.html.slim

```diff
@@ -0,0 +1,24 @@
+.content-group ng-if="::$ctrl.parent.isOwnershipListing($ctrl.parent.listing)"
+  h4.h-caps-lined
+    | Expected move-in date
+  p.t-small.caps
+    | {{ ::$ctrl.parent.listing.Expected_Move_in_Date | date : 'MMMM yyyy' }}
+
+.content-group
+  ng-include src="'listings/templates/listing/_what_to_expect.html'"
+
+.content-group ng-repeat="lotteryPreference in ::$ctrl.parent.listing.preferences" ng-if="!$ctrl.listingHasLotteryResults() && lotteryPreference.pdfUrl"
```

Comment:
I don't know what this actually looks like, but it looks like it's related to the lottery stuff? @callaghanc or @chrisdolendo do you know what this is or how to reproduce this for QA?

Reply:
hrm I asked larry about that for the lottery complete bug and I'm not sure that we know how to do that.